### PR TITLE
[TG-231] [fix] Fix hiding of widgets when user edits them on dashboar…

### PR DESCRIPTION
**Pull request by piton13**
…d page

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/123" target="_blank">ReviewNinja

File: src/app/dashboard/dashboard-editor/dashboard-editor.component.ts [hunk 1]
```diff
@@ -16,12 +16,10 @@ const selector = 'dashboard-editor';
     viewProviders: [DragulaService]
 })
 export class DashboardEditorComponent {
-    private dashboardService;
-    private router;
     private widgets;
     private initialData;
 
-    constructor(dashboardService: DashboardService, router: Router) {
+    constructor(private dashboardService: DashboardService, private router: Router) {
```
Comment (borys-ihnatyev) [inline src/app/dashboard/dashboard-editor/dashboard-editor.component.ts]: Actually you can omit assignment in constructor when you mark parameters with access modifier

Comment (piton13) [reply src/app/dashboard/dashboard-editor/dashboard-editor.component.ts]: fixed.

**Comment by disist** [comment]
Could you please update `ng2-dragula` to latest version with module support

File: src/app/dashboard/dashboard-editor/dashboard-editor.component.ts (hunk 1 above)
Comment (piton13) [inline src/app/dashboard/dashboard-editor/dashboard-editor.component.ts]: !resolved

**Pull request closed without merging**